libraries/AC_AutoTune/AC_AutoTune.h

```diff
@@ -16,14 +16,32 @@
   support for autotune of multirotors. Based on original autotune code from ArduCopter, written by Leonard Hall
   Converted to a library by Andrew Tridgell
  */
-
 #pragma once
 
 #include <AP_HAL/AP_HAL.h>
-#include <AC_AttitudeControl/AC_AttitudeControl_Multi.h>
+#include <AC_AttitudeControl/AC_AttitudeControl.h>
 #include <AC_AttitudeControl/AC_PosControl.h>
+#include <AP_Math/AP_Math.h>
+#include "AC_AutoTune_FreqResp.h"
+
+#define AUTOTUNE_AXIS_BITMASK_ROLL            1
+#define AUTOTUNE_AXIS_BITMASK_PITCH           2
+#define AUTOTUNE_AXIS_BITMASK_YAW             4
+
+#define AUTOTUNE_SUCCESS_COUNT                4     // The number of successful iterations we need to freeze at current gains
+
+// Auto Tune message ids for ground station
+#define AUTOTUNE_MESSAGE_STARTED 0
```

Comment:
I think maybe AUTOTUNE_MESSAGE_STARTED (and similar) are only used in AC_AutoTune.cpp so they remain back in the .cpp file.

Reply:
@rmackay All of these messages except AUTOTUNE_MESSAGE_SAVED_GAINS are used by methods only in the parent class.   AUTOTUNE_MESSAGE_SAVED_GAINS is used in the sub classes.  So I was thinking of keeping all of them together and in the AutoTune.h file.